src/mudlet.h

```diff
@@ -650,6 +652,32 @@ private slots:
 
     // Whether multi-view is in effect:
     bool mMultiView;
+
+    const QStringList mDefaultGames = {"3Kingdoms",
```

Comment:
Actually - can't this be a `static const QStringList` - then you can also use `QStringLiteral(...)` on each string...